integration-tests/smallrye-opentracing/src/test/java/io/quarkus/it/opentracing/OpenTracingTestCase.java

```diff
@@ -0,0 +1,317 @@
+package io.quarkus.it.opentracing;
+
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.common.mapper.TypeRef;
+
+@QuarkusTest
+public class OpenTracingTestCase {
+
+    static GenericContainer postgreSQLContainer;
+
+    @BeforeAll
```

Comment:
Does it also need an `@AfterAll` to cleanup the container?

Reply:
Testcontainers should stop it automatically (thanks to Ryuk container) but I can add one to force stopping the container eagerly